public/member_ops.js

```diff
@@ -180,9 +209,77 @@ function handle_members(event) {
         }
       });
     } else if (event.target === createFact) {
-      // Your code goes here!
+      var dataObject = {
+        text: factText.value,
+        likes: likes.value,
+      };
```

Comment:
It looks like the post data here doesn't match what the route expects, so it's returning 422 instead of creating a fact. You can check this against your swagger docs or look at the facts controller (in the `fact_params` method).